src/racer/matchers.rs

```diff
@@ -298,7 +300,7 @@ pub fn match_mod(msrc: Src, blobstart: usize, blobend: usize,
                 contextstr: filepath.to_str().unwrap().to_owned(),
                 generic_args: Vec::new(),
                 generic_types: Vec::new(),
-                docs: String::new(),  //TODO: fix
+                docs: String::new(),
```

Comment:
I realize this was added in a different PR, but this should probably hold an `Option<String>` and initialize to `None` here.